Game.cs

```diff
@@ -8,18 +9,500 @@ internal class Game
+                            }
+                            else if (player.steps >= 26 && player.steps <= 35)
+                            {
+                                Console.Clear();
+                                Console.WriteLine("You see a large mirror in the room, you can see your reflection in it");
+                            }
+                            else if (player.steps >= 36 && player.steps <= 49)
+                            {
+                                Console.Clear();
+                                Console.WriteLine("You see a large chest in the room, it seems to be locked");
+                            }
+                            else if (player.steps == 50)
+                            {
+                                Console.Clear();
+                                Console.WriteLine("You see a large door in front of you, it seems to be unlocked and light is shining through");
+                            }
+                            else
+                            {
+                                Console.Clear();
+                                Console.WriteLine("You see nothing of interest in the room");
+                            }
+                            break;
+                        //this case allows the player to move through the dungeon and spawns monsters at random intervals
+                        case "2":
+                            Console.Clear();
+                            Console.WriteLine("You walk forwards.");
+                            player.move();
+                            if (player.steps >= 5 && player.steps <= 15)
+                            {
+                                if (player.steps == 5)
+                                {
+                                    Console.WriteLine("You have now entered a new room");
+                                    player.addScore(100);
+                                }
+                                currentRoom = new Room("You are in a brightly lit room, with chandelliers and dining tables, food set out in an elegant fashion\n");
+                                //removes all items from room that the player has already picked up
+                                if (player.inventoryContents().Contains("Sword"))
+                                {
+                                    currentRoom.removeItem("Sword");
+                                }
+                                else if (player.inventoryContents().Contains("Shield"))
+                                {
+                                    currentRoom.removeItem("Shield");
+                                }
+                                else if (player.inventoryContents().Contains("Potion"))
+                                {
+                                    currentRoom.removeItem("Potion");
+                                }
```

Comment:
There are a few instances where this is done. This ideally should be put into the `Room` constructor and have the player inventory passed in as a parameter.

Alternatively, the pickup command could just not work for items the player has already picked up.